packages/prisma/schema.prisma

```diff
@@ -101,7 +101,7 @@ model EventType {
   userId            Int?
 
   profileId Int?
-  profile   Profile? @relation(fields: [profileId], references: [id], onDelete: Cascade)
```

Comment:
is it okay that personal event types now won't be deleted when user is removed from teams?